packages/markers/src/browser/problem/problem-tree-model.ts

```diff
@@ -74,6 +76,25 @@ export class ProblemTree extends MarkerTree<Diagnostic> {
         return 0;
     }
 
+    protected async refreshMarkerInfo(uri: URI): Promise<void> {
+        const id = uri.toString();
+        const existing = this.getNode(id);
+        const markers = this.markerManager.findMarkers({ uri });
+        if (markers.length <= 0) {
+            if (MarkerInfoNode.is(existing)) {
+                CompositeTreeNode.removeChild(existing.parent, existing);
+                this.removeNode(existing);
+                this.fireChanged();
+            }
+            return;
+        }
+        const node = MarkerInfoNode.is(existing) ? existing : this.createMarkerInfo(id, uri);
+        const children = this.getMarkerNodes(node, markers);
+        ProblemCompositeTreeNode.addChild(node.parent, node, markers);
+        this.setChildren(node, children);
+        node.numberOfMarkers = markers.length;
+    }
```

Comment:
This function appears to be a duplicate (at least in name) of a function in `marker-tree.ts`, and as far as I can tell, this version of the function is never called. Am I missing a call-site somewhere?

Reply:
The `problem-container` is binding the `ProblemTree`:
https://github.com/eclipse-theia/theia/blob/9ed915b80c587379a04b01f3e3c4486bd73d4d1a/packages/markers/src/browser/problem/problem-container.ts#L34-L39
The function call itself is here:
https://github.com/eclipse-theia/theia/blob/9ed915b80c587379a04b01f3e3c4486bd73d4d1a/packages/markers/src/browser/marker-tree.ts#L39

Reply:
It looks like you have deleted the duplicate implementation from the `ProblemTreeModel` class. 👍

Reply:
This method also had some code duplication from the method of the super class. With my last push I extracted some functionality in a new function to get rid of the duplicate code.